README.md

```diff
@@ -60,34 +69,53 @@ make -C ${ZKLLVM_BUILD:-build} assigner clang -j$(nproc)
 
 zkLLVM's workflow is as follows:
 
-1. Users wishing to prove some statement are supposed to implement an application in a language compatible with some frontend (C++ for now). This code will be compiled with a modified version of the `clang` compiler, which will output intermediate representation of the circuit. 
```

Comment:
@nemothenoone are statement and circuit the same thing? Why do we use both 'prove a statement' and 'generate a proof of the circuit'?

Reply:
Yup, it is the same thing. @SK0M0R0H would argue about this, but in context of this readme - it is the same.